sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,25 +1,51 @@
 # Release History
 
-## 1.8.0-beta.4 (Unreleased)
+## 1.8.0 (2023-02-02)
 
 ### Features Added
 
+- Added support for parsing space character in place of 'T' in RFC3339 DateTimes.
+- Added support for HTTP proxy servers, both unauthenticated and with Plain authentication.
```

Comment:
```suggestion
- Added support for HTTP proxy servers, both unauthenticated and with plain authentication.
```

Reply:
What is "plain" authentication? IIRC normally it's called "basic".